src/main/java/com/sergiofah/model/Category.java

```diff
@@ -13,12 +18,14 @@ public class Category implements Serializable {
     private Long id;
 
     @ManyToOne
+    @JsonIncludeProperties(value = {"id"})
```

Comment:
JSON formatting annotations shall not be in Entity classes. It's necessary to create DTO classes.